npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cloudgov-dashboard",
-  "version": "1.14.0",
```

Comment:
@adborden Should I be concerned that this was changing from `1.14.0` and not from `1.16.0`? If not, LGTM.